temba/orgs/tasks.py

```diff
@@ -45,3 +48,18 @@ def calculate_credit_caches():  # pragma: needs cover
 @nonoverlapping_task(track_started=True, name="squash_topupcredits", lock_key='squash_topupcredits')
 def squash_topupcredits():
     TopUpCredits.squash()
+
+
+@task(track_started=True, name='send_chatbase_logs')
+def send_chatbase_logs():  # pragma: needs cover
+    """
+    Send messages logs in batch to Chatbase
+    """
+    from temba.orgs.models import CHATBASE_API_KEY
+
+    for org in Org.objects.filter(config__icontains=CHATBASE_API_KEY):
+        cache_keys = cache.keys('org:%d:cache:chatbase_log:*' % org.id)
```

Comment:
Currently we are changing the value to `not_handled` parameter after the messages are handled, it's happening on Redis key before sending to Chatbase. So, working this way (just one Redis key to each organization), we have a problem, iterate in messages array to change this parameter, and it's very important to Chatbase like the documentation says. Could you give an idea about resolving that? Thanks.